app/src/main/java/org/eyeseetea/malariacare/domain/usecase/SaveSurveyAnsweredRatioUseCase.java

```diff
@@ -15,8 +15,18 @@ public SaveSurveyAnsweredRatioUseCase(
         mSurveyAnsweredRatioRepository = surveyAnsweredRatioRepository;
     }
 
-    public void execute(long idSurvey, GetSurveyAnsweredRatioUseCase.Callback callback) {
```

Comment:
UseCases should use AsynExecutor to execute and MainExecutor to invoke callbacks